### ISSUE TITLE
`<header>` use triggers Accessibility violation

### ISSUE THREAD

**Issue by luketa8** [grommet/grommet#7422]
In this [Example](https://unrivaled-bublanina-3a9bae.netlify.app/?path=/story/managing-child-objects--remove-all-action&globals=backgrounds.grid:!false) the `Document has more than one banner landmark` accessibility violation is triggered, possibly due to the use of the `<header>`. 

Currently 
> ![Screenshot 2024-11-08 at 4 41 13 PM](https://github.com/user-attachments/assets/5af4ba61-d70c-4f7b-8c46-6f8224fd7426)

Possible solution: The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable, or avoid usage of `<header>`altogether.
